ITEM_en.tsv

```diff
@@ -4830,15 +4830,15 @@ ITEM_20150612_005216	A propaganda to encourage the surrender sprayed by Kingdom
 ITEM_20150612_005217	Reaction Stone
 ITEM_20150612_005218	A reaction stone obtained to defeat the monsters in the Roxona market.
 ITEM_20150612_005219	Velas Grass
-ITEM_20150612_005220	
-ITEM_20150612_005221	Damaged Belongins
+ITEM_20150612_005220	This will finish off the smelly monsters.  If you leave without finishing them off, they'll continue propagating the nest.
+ITEM_20150612_005221	Damaged Belongings
 ITEM_20150612_005222	Seems the trail of times. There were fully filled with the damaged items.
 ITEM_20150612_005223	Monster Ink
 ITEM_20150612_005224	Such a black liquid obtained from the monsters.
 ITEM_20150612_005225	Monster Periderm Paper
 ITEM_20150612_005226	Such a thin film can replace the paper
 ITEM_20150612_005227	Blender
-ITEM_20150612_005228	
+ITEM_20150612_005228	Throw a monster in and make some juice!
```

Comment:
"A restraining device that allows the user to hold a monster still while he or she collects its extract."